src/lib/validation/sanitizers.ts

```
@@ -193,16 +192,18 @@
-    .replace(/[\u0000-\u0008\u000B\u000C\u000E-\u001F\u007F]/g, '')
-    // Normalize whitespace
-    .replace(/\s+/g, ' ');
+  return (
+    sanitizeInput(text, {
+      trimWhitespace: true,
+      stripHtml: true,
+      maxLength,
+      normalizeUnicode: true,
+    })
+      // Remove control characters except newlines and tabs
+      .replace(/[\u0000-\u0008\u000B\u000C\u000E-\u001F\u007F]/g, '')
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)